### PR TITLE
simulation tests for incentive

### PR DESCRIPTION
**Pull request by carloscaldas**
This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/2796)

Issue https://github.com/LBNL-UCB-STI/beam/issues/2788

File `incentives-ride_hail.csv` content below:
```
mode,age,income,amount
ride_hail,[0:10],[0:1000],20
```

### CODE REVIEW

**Comment by carloscaldas** [comment]
Test!

**Comment by carloscaldas** [comment]
Test!

File: src/test/scala/beam/sflight/SimulateIncentiveSpec.scala [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    val travelDistanceStats = injector.getInstance(classOf[org.matsim.analysis.TravelDistanceStats])
+    if (travelDistanceStats != null)
+      travelDistanceStats.close()
+    super.afterAll()
+  }
+
+  "SF Light" must {
+    "run 1k scenario with different incentives" taggedAs (Periodic, ExcludeRegular) in {
+      val lastIteration = 0
+      val avg1 = runSimulationAndReturnModeChoiceColumnSum(lastIteration, "incentives.csv")
```
Comment (wrashid) [inline src/test/scala/beam/sflight/SimulateIncentiveSpec.scala:35]: - withoutIncentiveNumberOfRidehailModeChoice
- is it average or sum?

File: src/test/scala/beam/sflight/SimulateIncentiveSpec.scala [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+      val avg2 = runSimulationAndReturnModeChoiceColumnSum(lastIteration, "incentives-ride_hail.csv")
```
Comment (wrashid) [inline src/test/scala/beam/sflight/SimulateIncentiveSpec.scala:36]: update name

**Comment by carloscaldas** [comment]
Build succeeded https://beam-ci.tk/job/beam-4ci/8483/
image

**Pull request merged**